source/val/Function.cpp

```diff
@@ -231,6 +245,11 @@ vector<BasicBlock*>& Function::get_blocks() { return ordered_blocks_; }
 const BasicBlock* Function::get_current_block() const { return current_block_; }
 BasicBlock* Function::get_current_block() { return current_block_; }
 
+BasicBlock* Function::get_pseudo_entry_block() { return &pseudo_entry_block_; }
```

Comment:
Agreed.  I've done it this way for consistency with the surrounding code for now.
I already raise this with Umar on a previous review, but it can be deferred.